src/main/java/com/scottlogic/librarygradproject/Exceptions/BookNotReservedException.java

```diff
@@ -0,0 +1,9 @@
+package com.scottlogic.librarygradproject.Exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
+public class BookNotReservedException extends RuntimeException{
+    public BookNotReservedException(Long bookId) {super("Book with id: " + bookId + " is not reserved"); }
```

Comment:
Should this be a 404? I get that it's because we can't "find" a reservation but the message doesn't really seem like a "not found" status